### ISSUE TITLE
Messages sent by bots fail WCAG AA contrast requirements

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#25808]
### Steps to reproduce

1. View a message sent by a bot, such as those in the This Week In Matrix room

#### What did you expect?

The text contrast ratio should meet accessibility requirements

#### What happened instead?

![Screenshot 2023-07-14 at 10-08-23 Element 10 This Week in Matrix (TWIM)](https://github.com/vector-im/element-web/assets/48614497/d602e8f2-579f-4327-8579-51bf81981f24)
![Screenshot 2023-07-14 at 10-56-41 WebAIM Contrast Checker](https://github.com/vector-im/element-web/assets/48614497/e3cc4e54-964d-4a77-b6c2-3431889f3f39)

### Operating system

NixOS unstable

### Browser information

Firefox 114.0.1

### URL for webapp

develop.element.io

### Application version

Element version: a84349f96979-react-eced103458e2-js-0b193f466562 Olm version: 3.2.14

### Homeserver

Not relevant

### Will you send logs?

No

**Comment by t3chguy** [comment]
This looks to be due to an `opacity: 0.6` present

Given it is even worse than that of staging this is a regression & blocker

![image](https://github.com/vector-im/element-web/assets/2403652/ebf35d15-1f2c-47f9-a393-ff79061c17ae)
